tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/SeqModelChecker.scala

```diff
@@ -46,6 +46,15 @@ class SeqModelChecker[ExecutorContextT](
   private val searchState: SearchState = new SearchState(params)
 
   override def run(): CheckerResult = {
+    // output an example of a run, if the context is satisfiable
+    def outputRun(): Unit = {
+      trex.sat(params.smtTimeoutSec) match {
+        case Some(true) =>
+          notifyExample(checkerInput.rootModule, trex.decodedExecution(), searchState.nRunsLeft)
+        case _ => ()
+      }
+    }
```

Comment:
Doesn't look like there's any reason for this function to be defined inside the `run` function. Maybe make it a private helper function? (Should help with performance, i'd think, also keeps `run` from adding unneeded lines?